docs/reference/supported-technologies.md

```diff
@@ -93,6 +93,7 @@ We support automatic instrumentation for the following messaging systems
 | Azure.Messaging.ServiceBus 7.0.0+ |
 | Azure Queue Storage {applies_to}`apm_agent_dotnet: ga 1.10` | Azure.Storage.Queues 12.6.0+ | [NuGet package](/reference/setup-azure-storage.md) |
 | Kafka {applies_to}`apm_agent_dotnet: ga 1.12` | See profiler documentation | [Profiler auto instrumentation](/reference/setup-auto-instrumentation.md) |
+| Kafka {applies_to}`apm_agent_dotnet: ga v1.34.2` | Confluent.Kafka 2.11.1 | [NuGet package](/reference/setup-kafka.md) |
```

Comment:
We need to think about the ga version for this as technically, this NuGet approach doesn't depend on the most recent version of the APM agent. We might want to map this to the nearest release after they introduced that observability package?